src/components/DisplayNames/DisplayNamesPropTypes.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+
```

Comment:
Not sure where to put this, but I think as a convention we should not use pascal case for things that are not namespaces or react components. If you look most other propTypes files are using camelcase.

Reply:
Hmmm... confused by this comment because a case-insensitive global search of `import prop` in Expensify.cash shows that almost all usages use PascalCase:

![image](https://user-images.githubusercontent.com/47436092/114233941-eab3d280-9932-11eb-8808-a5f2bc8d9b31.png)

The exceptions are when we have a propTypes file, we _export_ `{propTypes, defaultProps}` using camelCase. Were you referring to the file name? I think that's what you meant, so I changed the filename to use `camelCase`. It looks like we have been pretty inconsistent about this so far, so happy to standardize.

Reply:
Sorry @roryabraham I was referring to the file name.